app/src/controllers/client/user/UserController.ts

```diff
@@ -5,16 +5,21 @@ import { Gender as EntityGender, User as EntityUser } from '@entities/User'
 import { signUpSchema, updateUserSchema, userPasswordUpdateSchema } from '@client/user/schemas'
 import MailService from '@client/user/services/MailService'
 import { UnauthorizedError } from '@errors/ControllerErrors'
-import { convertDateObjectToOutboundDateString, convertDateStringToDateObject } from '@lib/Date'
+import {
+  convertDateObjectToOutboundDateString, convertDateStringToDateObject,
+  convertDateTimeObjectToDateTimeString,
+} from '@lib/Date'
+import { Reservation } from '@entities/Reservation'
 
 export type UserServiceInterface = {
   fetchUserWithReservationCountAndReviewCount(id: number)
     : Promise<(EntityUser & { reservationCount: number, reviewCount: number })>
   signUpUser(email: string, username: string, password: string, confirm: string): Promise<EntityUser>
   updateUser(id: number, lastNameKanji: string, firstNameKanji: string, lastNameKana: string,
-    firstNameKana: string, gender: Gender, birthday: Date): Promise<EntityUser>
-  updateUserPassword(id: number, oldPassword: string, newPassword: string, confirmNewPassword: string)
-    : Promise<EntityUser>
+    firstNameKana: string, gender: Gender, birthday: Date): Promise<User>
+  updateUserPassword(id: number, oldPassword: string, newPassword: string, confirmNewPassword: string): Promise<User>
```

Comment:
you changed the return value of update user and update user password to User, which is a req-res-type model